ActionPlans/Start-RecipientProvisioningInvestigation.ps1

```diff
@@ -0,0 +1,32 @@
+
+Function Search-RecipientObject
+{
+    param( 
+        [int][Parameter(Mandatory=$true)] $DaysToSearch,
+        [string[]][Parameter(Mandatory=$false)] $OperationsToSearch,
+        [string][Parameter(Mandatory=$false)] $userIds)
+      
+    $DaysToSearch=10
+    if (!([string]::IsNullOrEmpty($userIds)))
+    {
+        $UnifiedAuditLogs = Search-UnifiedAuditLog -StartDate (Get-Date).addDays(-$DaysToSearch) -EndDate (Get-Date) -Operations $OperationsToSearch - -UserIds $userIds -SessionCommand ReturnLargeSet 
```

Comment:
Please remove the unneccessary "-" from the command:
"$OperationsToSearch - -UserIds $userIds" should be "$OperationsToSearch -UserIds $userIds"